README.md

```diff
@@ -228,20 +228,114 @@ Now everything has been set up and OP-TEE is ready to be used.
 ---
 ### 4.3 Juno
 Juno has been supported in OP-TEE since mid October 2014.
-#### 4.3.1 Get the compiler and source
-Follow the instructions in the "4.1 Basic setup".
 
-#### 4.3.2 Build
+WARNING: 
+- OP-TEE linux driver type I support only (do not modify SHA1 ID)
+- OP-TEE linux driver type II support requires update:
++ Linux kernel version alignment (3.18-rc1) with QEMU/FVP (DMA_BUF API change)
++ Will need arch/arm/Kconfig patch(es) (i.e DMA_SHARED_BUFFER etc...)
+Temporary .config and dts patch files:
+config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+juno.dts.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+
+#### 4.3.1 Prerequisites 
++ To be able run this script you will need to install a couple of dependencies. On
+a Debian based system (Ubuntu, Mint etc.), you will at least need to install the
+following packages:
```

Comment:
* To be able *to* run ...
* you will at least need to install the following packages--> the following packages must be installed.